hummingbot/strategy/pure_market_making/pure_market_making_v2.pyx

```diff
@@ -152,6 +157,10 @@ cdef class PureMarketMakingStrategyV2(StrategyBase):
     def sizing_delegate(self) -> OrderSizingDelegate:
         return self._sizing_delegate
 
+    @property
+    def sizing_delegate_name(self) -> str:
+        return self._sizing_delegate.name
```

Comment:
Again, if you expect all sizing delegates to have a name, you must add it to the interface s.t. it's properly defined for all plugins of this type. The way you do it now is architecturally incorrect.